Add troubleshooting note on custom metrics (#10339)
DataDog/integrations-core: openmetrics/README.md

## Troubleshooting

> #### High custom metrics billing
> OpenMetrics configurations with generic wildcard values for the `metrics` option will have significant impact on custom metrics billing.
> 
> Consider using specific metric names or partial metric name matches for precise collection.
> 

Reviewer: ```suggestion
### High custom metrics billing

OpenMetrics configurations with generic wildcard values for the `metrics` option have significant impact on custom metrics billing.

Datadog recommends that you use specific metric names or partial metric name matches for more precise collection.
```
